### ISSUE TITLE
Refactor Particles component

### ISSUE THREAD

**Issue by Wojtazzzz** [Uczniowskie-Kolo-Informatyczne/uki-web-next.ts#6]
Component inside src/components/inc/Particles.tsx should be more generic.

**Comment by KrifiZ** [comment]
Wdym by saying it "should be more generic"? Are you suggesting that it should have all the properties of IParticlesProps or only the options? If it includes all the properties, then the particles init should be removed because it would be passed, right?

**Comment by KrifiZ** [comment]
> Component inside src/components/inc/Particles.tsx should be more generic. It should receive prop with TSParticle configuration (type of ISourceOptions). In addition particlesInit (local function) should be removed.

Wdym by saying it "should be more generic"? Are you suggesting that it should have all the properties of IParticlesProps or only the options? If it includes all the properties, then the particles init should be removed because it would be passed, right?

**Comment by Wojtazzzz** [comment]
> > Component inside src/components/inc/Particles.tsx should be more generic. It should receive prop with TSParticle configuration (type of ISourceOptions). In addition particlesInit (local function) should be removed.
> 
> Wdym by saying it "should be more generic"? Are you suggesting that it should have all the properties of IParticlesProps or only the options? If it includes all the properties, then the particles init should be removed because it would be passed, right?

Yes, we should create component which can be used in many places.
